src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -282,7 +307,16 @@ def get_prediction(model_path: str, dataset_evaluation: tf.data.Dataset) -> np.a
         dataset_evaluation = temp_dataset_evaluation.map(lambda _path, depthmap, targets: (depthmap, targets))
         del temp_dataset_evaluation
 
-        prediction_list_one = get_prediction(model_path, dataset_evaluation)
+        if RUN_ID == 'all':
+            prediction_list_one = []
+            for model_index, model_path in enumerate(model_paths):
+                print(f"Model {model_index + 1}/{len(model_paths)}")
+                prediction_list_one += [get_prediction(model_path, dataset_evaluation)]
+                print("Prediction made by model on the depthmaps...")
+            prediction_list_one = np.array(prediction_list_one)
+            prediction_list_one = np.mean(prediction_list_one, axis=0)
```

Comment:
Could we put this into a function?

```suggestion
            prediction_list_one = get_prediction_ensemble(model_path, dataset_evaluation)
```